README.md

```diff
@@ -3,13 +3,7 @@
 [![GitHub Build Status](https://github.com/cisagov/ansible-role-admiral/workflows/build/badge.svg)](https://github.com/cisagov/ansible-role-admiral/actions)
 [![CodeQL](https://github.com/cisagov/ansible-role-admiral/workflows/CodeQL/badge.svg)](https://github.com/cisagov/ansible-role-admiral/actions/workflows/codeql-analysis.yml)
 
-This is a skeleton project that can be used to quickly get a new
-[cisagov](https://github.com/cisagov) Ansible role GitHub project
-started.  This skeleton project contains
-[licensing information](LICENSE), as well as
-[pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for an Ansible role.
+Ansible Role that installs the Docker composition for [cisagov/admiral](https://github.com/cisagov/admiral).
```

Comment:
One more wording tweak to make this a complete sentence and also provide a brief explanation of what the Admiral is:
```suggestion
This is an Ansible role that installs the Docker composition for
[cisagov/admiral](https://github.com/cisagov/admiral), a Certificate
Transparency log harvester.
```